Common/GPU/OpenGL/GLMemory.h

```diff
@@ -124,6 +124,8 @@ class GLPushBuffer {
 		NextBuffer(numBytes);
 		*bindOffset = 0;
 		*buf = buffers_[buf_].buffer;
+		// Need to mark the allocated range used in the new buffer. How did things work before this?
+		offset_ = numBytes;
```

Comment:
I guess this was lost from #17480, so not all that old.  But we used to align to 4 bytes which seems sensible.  Though, not sure how often we'd push anything that wasn't already a multiple of 4...

-[Unknown]

Reply:
Hm, yeah seems so. I was hoping this would accidentally fix those weird OpenGL crashes on old Adreno maybe, but probably not then.